integration_tests/tests/integration/in_cluster.rs

```diff
@@ -32,8 +38,185 @@ struct InClusterJanusPair {
+    ///  - `JANUS_E2E_DIVVIUP_API_TOKEN` (Bearer token): API token with which to authenticate to the
+    ///    `divviup-api` instance at `JANUS_E2E_DIVVIUP_API_URL` as a member of account
+    ///    `JANUS_E2E_DIVVIUP_ACCOUNT_ID`.
+    ///  - `JANUS_E2E_DIVVIUP_ACCOUNT_ID` (UUID): Account ID in which to create a task.
+    ///  - `JANUS_E2E_LEADER_AGGREGATOR_ID` (UUID): ID of the aggregator to use as DAP leader in the
+    ///    task.
+    ///  - `JANUS_E2E_HELPER_AGGREGATOR_ID` (UUID): ID of the aggregator to use as DAP helper in the
+    ///    task.
+    ///  - `JANUS_E2E_COLLECTOR_CREDENTIAL_ID` (UUID): ID of the collector credential to use when
+    ///    collecting aggregate shares in the task.
+    ///  - `JANUS_E2E_COLLECTOR_CREDENTIAL_JSON`: JSON representation of the collector credential,
+    ///    including the auth token and the HPKE private key. Example:
+    ///
+    ///    {
+    ///      "aead": "AesGcm128",
+    ///      "id": 66,
+    ///      "kdf": "Sha256",
+    ///      "kem": "X25519HkdfSha256",
+    ///      "private_key": "uKkTvzKLfYNUPZcoKI7hV64zS06OWgBkbivBL4Sw4mo",
+    ///      "public_key": "CcDghts2boltt9GQtBUxdUsVR83SCVYHikcGh33aVlU",
+    ///      "token": "Krx-CLfdWo1ULAfsxhr0rA"
+    ///    }
+    async fn new_in_cloud(vdaf: VdafInstance, query_type: QueryType) -> Self {
+        let (
+            divviup_api_url,
+            divviup_api_token,
+            divviup_account_id,
+            leader_aggregator_id,
+            helper_aggregator_id,
+            collector_credential_id,
+            collector_credential,
+        ) = match (
+            env::var("JANUS_E2E_DIVVIUP_API_URL"),
+            env::var("JANUS_E2E_DIVVIUP_API_TOKEN"),
+            env::var("JANUS_E2E_DIVVIUP_ACCOUNT_ID"),
+            env::var("JANUS_E2E_LEADER_AGGREGATOR_ID"),
+            env::var("JANUS_E2E_HELPER_AGGREGATOR_ID"),
+            env::var("JANUS_E2E_COLLECTOR_CREDENTIAL_ID"),
+            env::var("JANUS_E2E_COLLECTOR_CREDENTIAL_JSON"),
+        ) {
+            (
+                Ok(divviup_api_url),
+                Ok(divviup_api_token),
+                Ok(divviup_account_id),
+                Ok(leader_aggregator_id),
+                Ok(helper_aggregator_id),
+                Ok(collector_credential_id),
+                Ok(collector_credential_json),
+            ) => (
+                divviup_api_url.parse().unwrap(),
+                divviup_api_token,
+                Uuid::parse_str(&divviup_account_id).unwrap(),
+                Uuid::parse_str(&leader_aggregator_id).unwrap(),
+                Uuid::parse_str(&helper_aggregator_id).unwrap(),
+                Uuid::parse_str(&collector_credential_id).unwrap(),
+                serde_json::from_str::<PrivateCollectorCredential>(&collector_credential_json)
+                    .unwrap(),
+            ),
+            _ => panic!("missing or invalid environment variables"),
+        };
+
+        let mut divviup_api = DivviupClient::new(
```

Comment:
I'm not sure why multiline suggestions aren't working, but another entirely-equivalent but slightly more concise option would be

```rust
let divviup_api = DivviupClient::new(divviup_api_token, RustlsConfig::<ClientConfig>::default())
    .with_default_pool()
    .with_url(divviup_api_url);

```